path.c

```diff
@@ -1274,7 +1274,7 @@ int is_mount_point_via_stat(struct strbuf *path)
 	strbuf_addch(path, '.');
 	if (lstat(path->buf, &st)) {
 		/*
-		 * If we cannot access the oarent directory, we cannot say
+		 * If we cannot access the parent directory, we cannot say
```

Comment:
Could I ask for my full name ("Johannes Schindelin") to be used in the Co-Authored-By: footer? Or alternatively, just drop that line, because I only authored the typo, not the fix :-)

Reply:
I think this came in with Github's "apply suggestion".